pkgs/top-level/all-packages.nix

```diff
@@ -39633,4 +39633,6 @@ with pkgs;
   udict = callPackage ../applications/misc/udict { };
 
   duden = callPackage ../applications/misc/duden { };
+
+  zf = callPackage ../tools/misc/zf { };
```

Comment:
Please sort this appropriately

Reply:
I have no clue how you want me to sort this... please give more precise feedback. The whole file is all sorts of not sorted so I figured might as well just append to the end.

Reply:
Sorting this 39k+-lines file is now a useless business.